autograph.testsignapk.yaml

```diff
@@ -0,0 +1,219 @@
+server:
```

Comment:
why is this new config file needed?

Reply:
Because creating the keyring on app start depends on gpg2 > 2.1 and this was a workaround for trying to get that installed on the test image

Reply:
We can remove it when either the testapksigner image circle uses a gpg2 > 2.1 or we switch to running the tests against containerized autograph